Khuzista.md

```diff
@@ -0,0 +1,3 @@
+. I want to make ecommerce page for a coffe shop.
+. I cannot scroll down on the page.
+. i do not undaerstand the problem. I think I did something wrong.
```

Comment:
-Create a cart page and try to add a product with price and buy button.
-Registration page, login page (2 points)
-Apply any relevant animation in the project (1 point)
-make sure the website is 100% accessibile (2 point)
make sure the website is 100% responsive (2 points)